packaging/docker/bootstrap_scripts/balloon.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+cwd=$(pwd)
+echo "# INSTALL & CONFIGURE BALLOON"
+echo "## INSTALL BUILD DEPENDENCIES"
+
+export BALLOON_VERSION=$(cat /srv/www/balloon/VERSION)
+if [ ${BALLOON_VERSION:0:1} -lt 2 ]
+then
+    echo "ERROR: Dockerfile requires a balloon version >= 2.0.0"
+    exit 1
+fi
+
+export DEBIAN_FRONTEND=noninteractive
+apt-get update
+apt-get --no-install-recommends install -y git npm make
```

Comment:
How does balloon get into the container (manually and build by a ci system)? A ci also injects the local build folder into the container?

Reply:
by default balloon is injected into the docker build context (https://docs.docker.com/engine/reference/builder/#usage) when building the image. from the build context it's copied into the container by the [Dockerfile](Dockerfile)